src/jquery.fancytree.js

```diff
@@ -3389,20 +3389,14 @@ $.extend(Fancytree.prototype,
 				node.li = document.createElement("li");
 				node.li.ftnode = node;
 
-				// if(aria){
-				// 	$(node.li).attr("role", "treeitem");
-				// }
 				if( node.key && opts.generateIds ){
 					node.li.id = opts.idPrefix + node.key;
 				}
 				node.span = document.createElement("span");
 				node.span.className = "fancytree-node";
 				if( aria && !node.tr ) {
-					$(node.span).attr("role", "treeitem");
```

Comment:
Putting on the li will allow level, posinset and setsize to be computed automatically by the browser. However, I am working on a fix for Chrome as it's adding descendant text from the child tree items into the auto-computed name. This is a very important bug that it brought to light. In the meantime, the buggy behavior is not bad in Chrome, so I think we should go with this.